source/features/dim-bots.css

```diff
@@ -8,8 +8,9 @@ Transitions are used to delay the "hover" status so it's not too annoying when m
 */
 .rgh-dim-bot.commit .commit-title,
 .rgh-dim-bot.commit .commit-meta,
-.rgh-dim-bot.commit .commit-links-cell summary,
-.rgh-dim-bot.commit .commit-links-cell .btn {
+.rgh-dim-bot.commit summary,
```

Comment:
If `summary` seems too broad, we can use `.signed-commit-badge` or something alike instead.